.travis.yml

```
@@ -1,21 +1,21 @@
 language: php
 
-dist: trusty
+dist: xenial
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env: PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 7.0
+    - php: 7.4
       env: PHPUNIT_TEST=1
```

```suggestion
    - php: 7.4
      env: PHPUNIT_TEST=1
    - php: 8.0
      env: PHPUNIT_TEST=1
```

If you want to go a step beyond, you could remove the `PHPUNIT_TEST` env var altogether and always run the build.

---

Updated